src/views/ActivityAppFeed.vue

```diff
@@ -21,7 +21,7 @@
 <template>
 	<NcAppContent class="activity-app">
 		<h1 class="activity-app__heading">
-			{{ t('activity', 'Your activities') }}
+			{{ t('activity', `${headingTitle}`) }}
```

Comment:
It is already translated

```suggestion
			{{ headingTitle }}
```